Code/Tools/FBuild/FBuildCore/WorkerPool/WorkerThread.cpp

```diff
@@ -176,46 +176,52 @@ void WorkerThread::WaitForStop()
     }
 
     // no local job, see if we can do one from the remote queue
```

Comment:
FBuild should always exist at this point. As with the other cases, this singleton not existing points to a problem elsewhere.

Reply:
Doesn't seem to be a problem now. Disregard?